package.json

```diff
@@ -3,17 +3,19 @@
   "packageManager": "yarn@3.2.0",
   "private": true,
   "workspaces": [
-    "packages/sdk",
-    "packages/ui",
     "packages/bots/fuse-liquidator-bot",
     "packages/bots/fuse-twap-bot",
-    "packages/monitors/oracle"
+    "packages/functions",
+    "packages/monitors/oracle",
+    "packages/sdk",
+    "packages/ui"
   ],
   "devDependencies": {
     "@istanbuljs/nyc-config-typescript": "^1.0.2",
     "@tsconfig/recommended": "^1.0.1",
     "@typescript-eslint/eslint-plugin": "^5.30.4",
     "@typescript-eslint/parser": "^5.30.4",
+    "netlify-cli": "^10.7.1",
```

Comment:
do we need this though?

Reply:
Convenient to have as no one needs to install 'netlify' globally on their machine for
`yarn dev:functions` which is using `netlify dev`.